packages/rollup/test/integration/fum/BUILD.bazel

```diff
@@ -7,7 +7,4 @@ js_library(
     srcs = ["index.js"],
     module_from_src = True,
     module_name = "fumlib",
-    # Don't allow deep imports under here,
-    # and give it the AMD name "fumlib", not "fumlib/index"
-    module_root = "index.d.ts",
```

Comment:
was this intended here? I could not find any use of the module_root attribute on any build path. Also d.ts seems out of place for a js library.

Reply:
yes, it's intended, I think there's a comment somewhere